### DISCUSSION TITLE
Destroy vs Hide banner

### DISCUSSION THREAD

**Discussion by takhimi** [Poing-Studios/Godot-AdMob-Android-iOS#101] [Q&A] [answered]
Hi. May I know what is the different between destroy and hide banner ? Usually I will just show the banner ads in menu screens and I will hide the banner when in game screen and show it again in game over screen,

Which should I use ? hide banner or destroy banner ? What is the different between the two ?

Thanks

**Comment by gumaciel** [comment, marked as answer]
Hi, the difference between Destroy and Hide is:

Destroy: will completely destroy the application's ad Banner, in other words the ad will no longer display and exist and you will need to call again the method `load_banner()`
Hide: will only hide the ad Banner so you can Show the banner instantly, without needing call `load_banner()`

You can use `destroy_banner()` if you prefer a greater performance gain (I believe the gain is very little, but depending on the cell phone it can be a good gain) and also don't have problem if it takes a few seconds for the Banner to appear again.

You can use `hide_banner()` if you prefer the banner to appear instantly and if it is loaded previously, use `show_banner()` to appear (`load_banner()` needs to be called beforehand)

---
Preferably I always use `destroy_banner()`, as I have no problem waiting a few seconds for the banner to appear again. But you can use whichever you like.

**Comment by takhimi** [comment]
Alright. Thank you for the clarification. Thanks again.
